qemu/tests/blockdev_full_backup_invalid_max_workers.py

```diff
@@ -0,0 +1,42 @@
+from virttest.qemu_monitor import QMPCmdError
+
+from provider import backup_utils
+from provider.blockdev_live_backup_base import BlockdevLiveBackupBaseTest
+
+
+class BlkFullBackupInvalidMaxWorkers(BlockdevLiveBackupBaseTest):
+
+    def do_full_backup(self):
+        max_workers = int(self.params["invalid_max_workers"])
+        extra_options = {"max-workers": max_workers}
+        cmd, arguments = backup_utils.blockdev_backup_qmp_cmd(
+            self._source_nodes[0],
+            self._full_bk_nodes[0],
+            **extra_options)
+        timeout = int(extra_options.pop("timeout", 600))
+        try:
+            self.main_vm.monitor.cmd(cmd, arguments)
+        except QMPCmdError as e:
+            error_msg = self.params["error_msg"]
+            if error_msg not in str(e.data):
+                self.test.fail(str(e))
+
```

Comment:
add else block in case the qmp cmd ran without any error

Reply:
Have updated, thanks zhenchao.